docs/connect-blockchain/multi-chain.mdx

```diff
@@ -399,8 +399,28 @@ import algosdk from "algosdk";
   Use code from the above Initializing Provider here
 */
 
-const privateKey = provider.request({ method: "eth_privateKey" });
+const privateKey = provider.request({ method: "private_key" });
 const passphrase = algosdk.secretKeyToMnemonic(Buffer.from(privateKey, "hex"));
 const keyPair = algosdk.mnemonicToSecretKey(passphrase);
 const address = keyPair.addr;
 ```
+
+### Polkadot
+
+For Polkadot, we will use the libraries `@polkadot/keyring` and `@polkadot/util-crypto` to create the Polkadot address.
+
+```js
+// Polkadot
+import { Keyring } from "@polkadot/api";
+import { cryptoWaitReady } from "@polkadot/util-crypto";
+
+/*
+  Use code from the above Initializing Provider here
+*/
+
+const keyring = new Keyring({ ss58Format: 42, type: "sr25519" });
+const privateKey = provider.request({ method: "private_key" });
```

Comment:
method will be `eth_private_key` here as well, you're using the `eip155` provider not the `other` provider

Reply:
Yes, I agree. We should be consistent here. Though, this works but should be from the eip155 provider to other providers. 

@Saviour1001, while you are at it, please update the page's wording to match that too.

Reply:
updated it

Reply:
Have we updated the example app to reflect the same?

Reply:
yep @shahbaz17 , its `eth_private_key` for the example as well

Reply:
Cool. Thanks for clarifying.